src/App.vue

```diff
@@ -49,34 +54,60 @@
 <script>
 import { getTubeLines, getStations, getPlatforms } from "./requests.js";
 
+// import Dropdown from './components/Dropdown.vue'
+import SingleDropdown from './components/singleDropdown.vue'
 
 export default {
   name: "App",
   data () {
     return {
       title: "Arrivals board",
+
+      lineLabel: "lines",
+      lineText: "Line: ",
+      linePlaceholder: "Choose a line",
       lines: [],
-      lineId: "",      
+      lineId: "",  
+
+      stationLabel: "stations",
+      stationText: "Station: ",
+      stationPlaceholder: "Choose a station",
       stations: [],
+      stationName: "",
       stopPoint: "",
+      
       timetable: [],
+      platformLabel: "platforms",
+      platformText: "Platform: ",
+      platformPlaceholder: "Choose a platform",
       platforms: [],
       platform: '',
+
       platformArrivals: [],
       timer: null,
       seconds: 30,
       isLoaded: false
     };
   },
-
+  props: ['tubeline'],
```

Comment:
Not sure what this is doing, is it definitely needed?

Reply:
Nope, removed it.